scripts/convert_toetsingskader_v1.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Shouldn't this script call the conversion util?

Reply:
For now we separated the conversion (Python) from the transformation (yq). If you so desire, we can call them from the conversion script.

Reply:
Added relevant files to gitignore.

Reply:
The conversion utils doesnt do anything related to a specific version (except for splitting files according to their version). It only cleans up some trailing whitespaces and parses dates (because pandas has an excellent date parser)